### ISSUE TITLE
[Item] "Box of Supplies" may be empty

### ISSUE THREAD

**Issue by Synful-Syn** [chromiecraft/chromiecraft#1230]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [ ] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [X] 30-39
- [ ] 40-49
- [ ] 50-59

### Current Behaviour

The item "Box of Supplies" (id 6827) may be empty.

### Expected Blizzlike Behaviour

Always contains something. TBC wowhead say the content is worth in average 2s 92c (of vendor value).

### Source

https://tbc.wowhead.com/item=6827/box-of-supplies

### Steps to reproduce the problem

1. Get "Box of Supplies" (id 6827).
2. Open it. Maybe you'll get nothing.

### Extra Notes

It has a list of possible rewards on aowow and wotlk.db but seems different than wowhead's loot table.
https://wowgaming.altervista.org/aowow/?item=6827
https://wotlkdb.com/?item=6827

Looking at the lists, the drop rate chance is very low on aowow and wotlk.db.

One of Thottbot's comment says it may contain a Swiftwind (item id 13038) but we don't get how rare it is.
https://tbc.wowhead.com/item=6827/box-of-supplies#comments:id=2675693

**Comment by jamadaha** [comment]
Tested on local server(AC fab0fc421)
The current behavior described is correct, it can drop nothing.
![image](https://user-images.githubusercontent.com/53914190/126074077-d5fa44ca-8d68-408a-b4b8-e60363b30fd5.png)

But I can find nothing in the source that guarantees a drop from it.

**Comment by Stump876** [comment]
I spawned and opened about 10. I ended up with a few potions, some herbs, and a couple agility scrolls, but no coin at all. I do not think the item loot table is wrong, it might not yield an item every time. I think the issue is it's not dropping any money reward. As noted in the comments in the link above "The reward is just a few silver most of the time." Supported by wowpedia https://wowpedia.fandom.com/wiki/Box_of_Supplies Some coin should be present in the box, even if an item is not. This also was an issue with fished up boxes and dropped bags not containing a coin reward. I believe this is valid.
